weasyl/moderation.py

```diff
@@ -608,8 +607,6 @@ def editcatchphrase(userid, otherid, content):
 
 
 def bulk_edit(userid, action, submissions=(), characters=(), journals=()):
-    web.header('Content-Type', 'text/plain')
```

Comment:
The good news is that in my uncommitted index, there's a `request.set_header_on_response()` call. After I merge this, I can go back and make massaction use it.